zerver/views/messages.py

```diff
@@ -308,6 +313,23 @@ def _by_search_tsearch(self, query, operand, maybe_negate):
         cond = column("search_tsvector").op("@@")(tsquery)
         return query.where(maybe_negate(cond))
 
+def highlight_match_tag_pos(text, pos, loc, tag_open, tag_close):
+    # type: (AnyStr, int, Tuple[int, int], AnyStr, AnyStr) -> Tuple[int, int]
+
+    (offset, length) = loc
```

Comment:
Yeah, but it was done that way in the other highlight functions, so I decided to keep them. Thanks for letting me know though - I'm not a python professional yet.